src/collector/pending.js

```diff
@@ -12,6 +12,17 @@ const CHECK_INTERVAL = 120; // seconds
 // thresholds for pending tasks.
 const MIN_CHECK_AGE = 300; // seconds
 
+// useless test provisioners
+const IGNORE_PROVISIONERS = [
+  'test-provisioner',
+  'dummy-test-provisioner',
```

Comment:
Yeah, ideally we should move everything to use `dummy-test-provisioner` for all the stupid stuff...